apps/passport-server/src/services/generic-issuance/pipelines/CSVPipeline/makePODTicketPCD.ts

```diff
@@ -0,0 +1,46 @@
+import { ArgumentTypeName, SerializedPCD } from "@pcd/pcd-types";
+import { PODTicketPCD, PODTicketPCDPackage } from "@pcd/pod-ticket-pcd";
+import { traced } from "../../../telemetryService";
+import { rowToTicket } from "./makeTicketPCD";
+
+export async function makePODTicketPCD(
+  inputRow: string[],
+  eddsaPrivateKey: string,
+  requesterEmail: string | undefined,
+  requesterSemaphoreId: string | undefined,
+  pipelineId: string
+): Promise<SerializedPCD<PODTicketPCD> | undefined> {
+  return traced("", "makePODTicketPCD", async () => {
+    if (!requesterEmail || !requesterSemaphoreId) {
+      return undefined;
```

Comment:
interesting to return `undefined` here. hopefully that's handled properly by the callers.